### PR TITLE
Badge#61

### PR DESCRIPTION
**Pull request by emdrayallday**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/components/Badge/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const badgeClasses = clsx(
+        'brew-Badge',
+        `brew-Badge--anchorFrom-${anchorFrom}`,
+        `brew-Badge--color-${color}`,
+        { 'brew-Badge--isCircular': isCircular },
+        { 'brew-Badge--inline': Boolean(!children) },
```
[the remark this answers is not in the document]
Comment (emdrayallday) [reply src/components/Badge/index.tsx]: Yeah I wasn't sure about that when I did it, I was thinking children would be falsey and not a true boolean which is why I put it there. Fixed it though.

**Comment by joshrcook** [comment]
@emdrayallday @lawren  One more question about this particular PR.  I know that there are instances where we have notification "dots".  Is the badge supposed to account for that use case as well?  It does in the Ant Design system, but of course we aren't following that to a tee...  Just thought I'd ask before we pulled this in as is.

https://ant.design/components/badge/

File: src/components/Badge/test.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { shallow } from 'enzyme';
+import Badge from './index';
+
+
+describe('Badge', () => {
+
+
+    it('Renders correctly', () => {
+        const tree = renderer.create(<Badge><div>child</div></Badge>).toJSON();
+
```
Comment (alyssadee) [inline src/components/Badge/test.tsx]: Should there be an extra space here, and also on line 32.

Comment (emdrayallday) [reply src/components/Badge/test.tsx]: I was thinking no, but I went ahead and added double spacing to the test file for consistency sake.

**Comment by emdrayallday** [comment]
> @emdrayallday @lawren One more question about this particular PR. I know that there are instances where we have notification "dots". Is the badge supposed to account for that use case as well? It does in the Ant Design system, but of course we aren't following that to a tee... Just thought I'd ask before we pulled this in as is.
> 
> https://ant.design/components/badge/

Yes this is possible, having the `isCircular` prop true and passing an empty string to the `content` prop

**Pull request merged**